inst/CITATION

```diff
@@ -8,3 +8,38 @@ bibentry(bibtype="Article",
          year=2006,
          url="https://igraph.org")
 
+note <- sprintf("R package version %s", meta$Version)
+# https://github.com/cran/lidR/blob/f0dae39007c9d174f6e1962ff236fd8826f1501d/inst/CITATION#L21
+year <- format(Sys.Date(), "%Y")
+authors <- meta$`Authors@R`
+authors <- utils:::.read_authors_at_R_field(authors)
+authors <- Filter(function(e) {
+  !(is.null(e$given) && is.null(e$family)) && !is.na(match("aut", e$role))
+}, authors)
+authors <- format(authors, include = c("given", "family"))
+
+# Step 1: Get text citation and replace there the abb.
+entry_for_txt <- format(bibentry(
+  bibtype = "Manual",
+  title = "{igraph}: Network Analysis and Visualization in R",
+  author = authors,
+  year = year,
+  note = note,
+  doi = "10.5281/zenodo.7682609",
+  url = "https://CRAN.R-project.org/package=igraph"
+), style = "text")
+
+txt <- gsub("Horvát S", "Horvát Sz", entry_for_txt)
+
+aut_new <- sub("Szabolcs", "{\\\\relax Sz}abolcs", authors)
```

Comment:
This came back from CRAN:

```
Problems when formatting CITATION entries:
  x:1: unexpected END_OF_INPUT '\relax'
```

Do we really need four backslashes here? Would `"{Sz}abolcs"` work?

Reply:
No, `{Sz}` does not achieve the intended effect.

However, I feel that I already caused too much trouble by even bringing this up. If there's no easy solution, I don't want to stand in the way of removing this feature.  What concerns me is that Maëlle already put a lot of work into making it work ...

Reply:
Thanks, no worries. We can try again with the next release. I now understand the four backslashes too.

Unfortunately, I can't replicate the error locally. It could require a specific locale.